Mobl_ios_Callisto-master/MealTrackingPilotTests/PilotLoginClientTests.swift

```diff
@@ -244,36 +250,36 @@ fileprivate final class MockAccessCredentialsLoginClient: AccessCredentialsLogin
         }
     }
 
-    fileprivate func handleRedirectURL(url: URL, options: [UIApplicationOpenURLOptionsKey : Any]) -> Bool {
+    fileprivate func handleRedirectURL(url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
         return true
     }
 
-    fileprivate func refreshLogin(completion: ((Result<AccessCredentials>) -> Void)?) {
+    fileprivate func refreshLogin(completion: ((Result<AccessCredentials,Error>) -> Void)?) {
         login()
     }
 }
 
 fileprivate final class MockResponseAPIClient: APIClient {
     var requestSent: URLRequest?
-    var mockResult: Result<Data?>
+    var mockResult: Result<Data?,Error>
 
-    init(mockResult: Result<Data?>) {
+    init(mockResult: Result<Data?,Error>) {
         self.mockResult = mockResult
-        super.init(session: .shared, decoder: JSONDecoder.CallistoJSONDecoder())
+        super.init()
     }
 
-    fileprivate override func sendRequest(_ request: URLRequest, completion: ((Result<Data?>) -> Void)?) {
+    fileprivate func sendRequest(_ request: URLRequest, completion: ((Result<Data?,Error>) -> Void)?) {
```

Comment:
change ((Result<Data?,Error>) -> Void) to defaultRequestCompletion

Reply:
Done